core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/sources/csv/StageDataStep.java

```diff
@@ -289,6 +291,9 @@ public DataType targetToSource( Integer value ) {
       tree.setSelectedRows( new int[] { 0 } );
     }
     tree.update();
+
+    // to focus first tab-able element
+    wizardDialog.center();
```

Comment:
If focusing the first tabbable element is the only goal, then `doAutoFocus()` should be called instead.

Reply:
Created PR https://github.com/pentaho/pentaho-commons-xul/pull/230